### ISSUE TITLE
Streams not showing in admin panel 

### ISSUE THREAD

**Issue by viralpatel380** [illuspas/Node-Media-Server#357]

**Comment by hthetiot** [comment]
If not showing did you check if it actualy started ?

**Comment by mlsgrnt** [comment]
Yes. Attempting to get HLS or DASH streams results in a 404 error and trying to get the flv results in an error in the console. As far as I can tell this is a clone of #319
